CONTRIBUTING.md

```diff
@@ -82,6 +82,20 @@ $ ninja install
 $ meson test
 ```
 
+## MacOS support (experimental)
+Most of the dependencies can be installed via [brew](brew.sh)
+```bash
+cd Gameeky
+brew install meson pkg-config libadwaita cmake gtksourceview5 desktop-file-utils gstreamer
+meson setup _test --prefix=$PWD/dist -Dwebkit=disabled
```

Comment:
Since this line is aimed at installing Gameeky, maybe `meson setup _build -Dwebkit=disabled` instead?

Also, not sure how macOS handles this, but is there something like `~/.local/` where Gameeky be installed without messing with the system?